CHANGES.md

```diff
@@ -1,5 +1,6 @@
 # Changelog
-
+### Room Default Configuration
+  support list-multi
```

Comment:
Please put this under the `## 5.0.0 (Unreleased)` heading instead of making your own heading.

Reply:
thanks@jcbrand